http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/AutopauseMonitor.scala

```diff
@@ -35,7 +35,9 @@ class AutopauseMonitor[F[_]](
   val process: Stream[F, Unit] =
     (Stream.sleep[F](config.autoFreezeCheckInterval) ++ Stream.eval(
       autoPauseCheck
-        .handleErrorWith(e => logger.error(e)("Unexpected error occurred during auto-pause monitoring"))
+        .handleErrorWith(e =>
+          logger.error(e)("Unexpected error occurred during auto-pause monitoring") >> F.raiseError[Unit](e)
```

Comment:
We are gathering more info on this issue as it may be happening across Leo. Also, we have a metric alert when this happens.

Reply:
having this change will still be useful tho...